swagger2.0/oic.r.rule.inputcollection.swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OCF Rule Input Collection",
+    "version": "20190910",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/RuleInputCollectionResURI?if=oic.if.ll": {
+      "get": {
+        "description": "Collection of Links to the Resources (i.e., Rule Inputs) that contain the Properties whose values are evaluated as part of the Rule Expression.",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the rule input Links.",
+            "x-example":
+            [
+              {
+                "anchor": "timeperiod",
+                "href": "/mytimeperiodresource",
+                "rel": ["ruleinput"],
+                "rt":   ["oic.r.time.period"],
+                "if":   ["oic.if.a","oic.if.baseline"],
+                "p":    {"bm": 3},
+                "eps": [
+                    {"ep": "coaps://[fe80::b1d6]:1111"}
+                ]
+              }
+            ],
+            "schema": { "$ref": "#/definitions/slinklist" }
+          }
+        }
+      }
+    },
+    "/RuleInputCollectionResURI?if=oic.if.baseline": {
+      "get": {
+        "description": "Collection of Links to the Resources (i.e., Rule Inputs) that contain the Properties whose values are evaluated as part of the Rule Expression.",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the rule input Links.",
+            "x-example":
+            {
+              "rt": ["oic.r.rule.input"],
+              "if": ["oic.if.ll","oic.if.baseline"],
+              "n":  "My Rule Inputs",
```

Comment:
Should we add "rts" here as required by the CR?

Reply:
Yes as the CR does note this as mandatory.

Reply:
Yes as the CR does note this as mandatory.